ansible/ansible-version.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
```

Comment:
Try `./play --check --limit datacollector.infra.ooni.io deploy-pipeline.yml` -- assertion using `hosts: localhost` will be skipped in that case.